proto/topo.proto

```diff
@@ -62,6 +63,7 @@ message Node {
     CISCO_XRD = 12;
     CISCO_E8000 = 13;
     LEMMING = 14;
+    ALPINE = 15;
```

Comment:
the type enum is deprecated, this should be removed

Reply:
Ack